centreon/src/Core/ResourceAccess/Infrastructure/API/UpdateRule/UpdateRulePresenter.php

```
@@ -23,14 +23,35 @@
 use Core\Infrastructure\Common\Api\DefaultPresenter;
 use Core\ResourceAccess\Application\UseCase\UpdateRule\UpdateRulePresenterInterface;
 
 class UpdateRulePresenter extends DefaultPresenter implements UpdateRulePresenterInterface
 {
     public function presentResponse(ResponseStatusInterface $response): void
     {
+        if ($response instanceof ErrorResponse && ! is_null($response->getException())) {
+            ExceptionLogger::create()->log($response->getException(), $response->getContext());
+        } elseif (
+            ($response instanceof ConflictResponse || $response instanceof InvalidArgumentResponse)
+            && isset($context['exception'])
```

PHPMD (2.15.0)

Avoid unused local variables such as '$context'. (Unused Code Rules)

(UnusedLocalVariable)